### ISSUE TITLE
Linux sync preview crashes when editing existing files

### ISSUE THREAD

**Issue by moritzheiber** [jotta/jotta-cli-issues#132]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
```
$ jotta-cli version
jotta-cli version 0.9.40199`
$ jottad version
0.9.40199
```

**Description of problem:**
When editing an existing file inside a  "synced" folder the `jottad` process dies/ends itself.

**Expected:**
The daemon doesn't crash and syncs my files continuously.

**jotta-cli status (`jotta-cli status`):**
```
Error: Could not connect to jottad. Is it running on 127.0.0.1:14443 ?
```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
```
pid:2930031 2021/02/10 13:38:52 "/some/path/Jottacloud/.jottaclouddownload/a62e9c514d756bce7018bf639affbf21": RENAME
pid:2930031 2021/02/10 13:38:52 "/some/path/Jottacloud/some-folder/document.md": RENAME
panic: NOPENOPE. two fsnotify.rename events in a row, but the src still exists?!?!?!

goroutine 16 [running]:
git.jotta.us/jotta/desktop/client/pkg/events/fswatch.(*FSWatch).loop(0xc0004c84e0)
	/builds/jotta/desktop/client/pkg/events/fswatch/fswatch_linux.go:107 +0x68b
created by git.jotta.us/jotta/desktop/client/pkg/events/fswatch.New
	/builds/jotta/desktop/client/pkg/events/fswatch/fswatch_linux.go:64 +0x10b
```
The daemon is gone/crashes after the last line, there's no debug log being submitted or collected.

_Note: I've redacted/changed certain file/folder names in order to preserve my privacy._

**Additional info:**
I'm running this on Ubuntu 20.04.2 LTS with all the latest upgrade installed. Filesystem is plain ext4 through on a LVM device adressed through Device Mapper.

**Comment by Kimbsen** [comment]
Thanks for the report!

We'll fix this for the next release

**Comment by Kimbsen** [comment]
This should be resolved in version 0.9.40946 unstable.
